### ISSUE TITLE
Google Analytics Dashboards

### ISSUE THREAD

**Issue by frankchen07** [gitcoinco/data-ops#12]
- [x] We need some data views on friction points in the Gitcoin flow for funders - is there a particular stage that they drop off at? 

The funnel looks something like:

1. goes to funder form
2. dismisses metamask / 'how to do this model'
3. inserts github url
4. submits form
5. submits metamask tx
6. bounty is posted

- [ ] breakdown of our new users day over day by channel (PPC, codefund, adwords, organic, SEO, etc)

**Comment by frankchen07** [comment]
Summary of current findings / documenting late night GA talks with @PixelantDesign (@vs77bb @owocki)

Google Analytics has some drawbacks when it comes funnel analysis:

1. Google Analytics currently doesn't support funnels that mix a combination of destinations (URLs) and events (button clicks or entering text on a form).

Therefore the funnel below is categorized as so:

1. goes to funder form (url)
2. dismisses metamask / 'how to do this model' (button)
3. inserts github url (form)
4. submits form (button)
5. submits metamask tx (untracked)
6. bounty is posted (url)

If we wanted a funnel analysis, Google Analytics would only support seeing users move from the funder form to the "bounty is posted" stage. Arguably, this is still useful, but if the goal is to see how and where funders are abandoning the form, GA is not our tool. If we're only talking about step 1 to step 6, we have that information in HotJar (example below).

<img width="1160" alt="screen shot 2018-08-23 at 3 22 44 am" src="https://user-images.githubusercontent.com/7516920/44568812-2e7b4e80-a72d-11e8-986c-abde665887e8.png">

> Contributor FTUX

For the contributor FTUX, it generally seems that of the sources that our contributors are coming from, 68% go directly to the issue explorer.

Of the remaining users who mostly end up on the contributor landing page, 85% exit the onboarding funnel. Of the 15% who don't exit, the majority of them directly ping the explorer. A very small number of users actually go through the traditional contributor FTUX (`onboard/contributor` and setting up Github, MetaMask, etc).

Looking at the volume of users on the goal flow intuitively tells us that the explorer is the location that most users gravitate towards. Applying the "new visitor" filter yields similar results with smaller absolute numbers.

<img width="1520" alt="screen shot 2018-08-23 at 23 35 26" src="https://user-images.githubusercontent.com/7516920/44568826-3d620100-a72d-11e8-8691-9f9925d75f3b.png">

What are users doing on the explorer if they're new and haven't connected their Github or Metamask? My hypothesis is that they're likely exploring and connecting with those two sources later.

**Could it be worth thinking about changing the order of the onboarding flow to come after taking a look at the explorer (this allows users to anchor themselves to an interesting issue that can motivate them to set things up)?**

> Funder FTUX

It seems that 0 funders have gone through the funder onboarding flow (prefixed by `onboard/funder`) - most funders seem to go directly to the `bounty/new` page, which implies they set up Github, Metamask, and their avatar in different ways.

For the Funder FTUX, this is where GA tracking on the funder page splits off from HotJar.

GA tracks 1-4, where as HotJar tracks just 4-5.

1. `onboard/funder`
2. `onboard/funder/metamask`
3. `onboard/funder/avatar`
4. `bounty/new`
5. `issue/[newly_created_issue]`

4-5 doesn't tell us anything about the friction points on the funder form itself, nor about where and how funders are actually getting their information setup. Essentially, funders haven't been using the FTUX (the way it was intended for). 

This might speak to the fact that many funders are being obtained by word of mouth, and not necessarily through marketing or other means that require a FTUX process. 

**I think this raises the priority of getting my hands on the HotJar data to see if there's any way we can determine the funder form friction points.**

**Comment by owocki** [comment]
i agree GA has some limitations here.  in the past, i've used mixpanel for funnel analysis, and it is quite good at that..  does metabase have any sort of funnel analysis?  i'd love to not have to introduce another tool

@vs77bb can we get frank the hotjar data?

**Comment by PixelantDesign** [comment]
Thanks for documenting @frankchen07 this is great!

**Comment by frankchen07** [comment]
@owocki - I've also used Mixpanel in the past (it's a bit unwieldy as well, but similar to GA, it gets the job done in simple scenarios). I also don't like tool creep. 

I've upped the priority of getting access and exploring HotJar data in parallel with some Metabase build outs for Scott, Metabot, and marketing.

**Comment by vs77bb** [comment]
@PixelantDesign What do you think about this change that Frank proposes? 

> Could it be worth thinking about changing the order of the onboarding flow to come after taking a look at the explorer (this allows users to anchor themselves to an interesting issue that can motivate them to set things up)? 

What if we allow a new user to see 15 issues on the Issue Explorer, but require someone to sign-up using the FTUX if a) the user wants to see more issues or b) the issue clicks on Issue Details? 

Agree with keeping this issue closed, though I think we could open up a new issue for this if we think it's useful. Not sure what the priority would be on making this happen. 🤔
